static/bidder-info/alkimi.yaml

```diff
@@ -0,0 +1,15 @@
+adapters:
+  alkimi:
```

Comment:
no need to add `adapters` and `alkimi` key in bidder-info yaml. Could start with endpoint

Reply:
fixed